sp041_20191205T1757.cdl

```diff
@@ -332,4 +332,46 @@ data:
  PLATFORM_MODEL = "Spray" ;
 
  WMO_IDENTIFIER = "4801948" ;
+
+group: CTD {
+
+  // group attributes:
+  		:long_name = "CTD sensor" ;
+  		:type = "CTD" ;
+  		:type_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
```

Comment:
This is the NVS collection for instrument type - https://vocab.nerc.ac.uk/collection/L05/current/
To be adopted.